modules/ROOT/pages/visual-tour/index.adoc

```diff
@@ -48,9 +48,18 @@ See xref:query/visual-tour.adoc#copilot[Query co-pilot] and xref:explore/explore
 
 image::enable-copilot.png[]
 
+==== Settings for GDS sessions
+
+include::partial$EAP.adoc[]
+
```

Comment:
Further, you can enable/disable the feature for the whole organization

Reply:
Nice, how would you do that? Org-level setting?

Reply:
yes there are two buttons on for `Modify Limits` and another to `Activate/Deactivate` the feature